src/main_menu.cpp

```diff
@@ -243,6 +242,77 @@ std::vector<std::string> main_menu::load_file( const std::string &path,
     return result;
 }
 
+/* Calculate date of Easter - a direct implementation of the "Anonymous Gregorian" aka Meeus/Jones/Butcher Gregorian algorithm, given here:
+    https://en.wikipedia.org/wiki/Computus#Anonymous_Gregorian_algorithm */
```

Comment:
I think this makes the comment prettier. If you don't agree, you can ignore this.
```suggestion
/*
 * Calculate date of Easter - a direct implementation of the "Anonymous Gregorian"
 * aka Meeus/Jones/Butcher Gregorian algorithm, given here:
 * https://en.wikipedia.org/wiki/Computus#Anonymous_Gregorian_algorithm
 */
```